src/include/network/lan/complex.rb

```diff
@@ -285,6 +285,25 @@ def enableDisableButtons
       nil
     end
 
+    # Updates the udev rule of the current Lan Item, adding the dev_port and
+    # using the bus_id instead ot the mac address.
+    def use_udev_rule_for_bonding!
```

Comment:
The two options you are given in the first sentence was what made me also doubt about which name method to use, because it currently use ReplaceItem for both (dev_port and mac_address) so basically it will update_or_create the dev_port attribute in the udev_rulle (if exists) with the read value and will replace the mac_address with the bus_id.